src/app.py

```diff
@@ -87,9 +87,19 @@ def bootstrap(_return=0):
                         logger.info('Load test is stopped.')
 
                         logger.info('Downloading reports...')
+                        time.sleep(4)
```

Comment:
why do we need this sleep?

Reply:
because locust has a delay of report data collection and report generation. We may even need to provide an option for that in the future